packages/app-studio-toolkit/src/actions/actionsConfig.ts

```diff
@@ -1,11 +1,31 @@
-import { workspace } from "vscode";
+import { workspace, WorkspaceConfiguration } from "vscode";
+import { uniqWith, isEqual } from "lodash";
 
 const key = "actions";
 
-export const get = (): string[] => {
-  return workspace.getConfiguration().get(key, []);
+const addActions = (actions: any[], config: WorkspaceConfiguration): void => {
+  const configActions = config.get<any[]>(key, []);
+  actions.splice(actions.length, 0, ...configActions);
 };
 
-export const clear = (): Thenable<void> => {
-  return workspace.getConfiguration().update(key, []);
+export const get = (): any[] => {
+  const actions: any[] = [];
+
+  workspace.workspaceFolders?.forEach((wsFolder) => {
```

Comment:
it is better to use lodash's `forEach` because the built-in high order functions in JS are very limited and insufficient and if we use both them and lodash's style we will inconsistent code style. 

So might as well use the style of the larger/more complete set of utilities (lodash)